content/chat/rooms/index.textile

```diff
@@ -104,6 +106,8 @@ Releasing a room may be optional for many applications. If you have multiple tra
 blang[javascript].
   Once "@rooms.release()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat_js.Rooms.html#release has been called, the room will be unusable and a new instance will need to be created using "@rooms.get()@":#create if you want to reuse it.
 
+  Note that calling @rooms.release()@ will cause any unresolved calls to @rooms.get()@ reject.
```

Comment:
This doesn't quite read right at the end. Do you mean `to @rooms.get()@ be rejected.`?

Reply:
Done!